### PR TITLE
Code Review #1

### PR DESCRIPTION
**Pull request by azdrenymeri**

### CODE REVIEW

File: stylesheets/styles.css [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    border-top: 1px solid #d9d9d9;
+    padding-top: 15px;
+}
+.byline{
+    font-size: 14px;
+    line-height: 21px;
+    color: #555;
+    text-transform: uppercase;
+    margin-top: 4px;
+    font: bold 17px/21px "Roboto Condensed",sans-serif;
```
Comment (heyapricot) [inline stylesheets/styles.css:25]: You are overwriting the `font-size` property

File: stylesheets/styles.css [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    width: 100%;
+    height:auto;
+}
+.block-title {
+    color: rgba(3,3,3,.4);
+    font: bold 16px/16px "Roboto Condensed",sans-serif;
+    text-transform: uppercase;
+    padding-bottom: 11px;
+}
+.category {
```
Comment (heyapricot) [inline stylesheets/styles.css:224]: Duplicate selector. You are using it on line 136

File: stylesheets/styles.css [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+.related li {
+    /* border-top: 1px solid #d9d9d9; */
+    /* border-bottom: 1px solid #d9d9d9; */
+    padding: 10px 0;
+    list-style: none outside none;
+}
+.related li a{
+    font-size: 17px;
+    line-height: 20px;
+    font: bold 16px/20px "Roboto Condensed",sans-serif;
```
Comment (heyapricot) [inline stylesheets/styles.css:241]: This line overwrites the previous two lines

**Review by heyapricot — CHANGES_REQUESTED** [review]
Almost there! Needs a bit more of work
Hello! 👋🏼 You are so close! Please address the comments, and after that submit a new code review request. Thanks! 

Comments:

- The width of your version is different from the original. Make it match the width of the original site
- You are missing the Editor's pick, The Featured Slideshows and the rest of the sections (US, Business, etc)
- Align the links from the footer in the same way that the original page. Please, also include the social media icons. 

Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

**Comment by azdrenymeri** [comment]
@heyalvaro thanks a lot for the code review. 👍 
And also for the tips.

**Pull request closed without merging**